src/Component/InformationModal/InformationModal.tsx

```diff
@@ -0,0 +1,360 @@
+import {
+  exampleConfig as LayerSourceConfigExample
+} from './information/layer/SourceConfigInformation';
+import {
+  exampleConfig as LayerFeaturesExample
+} from './information/layer/FeaturesInformation';
+import {
+  exampleConfig as UserProviderDetailsExample
+} from './information/user/ProviderDetailsInformation';
+import {
+  exampleConfig as UserDetailsExample
+} from './information/user/DetailsInformation';
+import {
+  exampleConfig as UserClientConfigExample
+} from './information/user/ClientConfigInformation';
+import { JSONSchema7 } from 'json-schema';
+import { getDocDataforTable, getDocDescription } from './information/helpers';
+import { CopyOutlined } from '@ant-design/icons';
+
+import './InformationModal.less';
+
+interface InformationModalProps {
+  dataField?: string;
+  entity?: string;
+  isModalOpen: boolean;
+  setIsModalOpen: (value: boolean) => void;
+  schema: JSONSchema7;
+}
+
+export const InformationModal: React.FC<InformationModalProps> = ({
+  dataField = '',
+  entity = '',
+  isModalOpen = false,
+  setIsModalOpen,
+  schema = undefined
+}) => {
+
+  const tmpFlat = [];
+
+  const docTableColums = (idCode: string): ColumnsType  => [
+    {
+      title: 'Property Name',
+      dataIndex: 'propertyName',
+      key: 'propertyName' + idCode
+    },
+    {
+      title: 'Description',
+      dataIndex: 'description',
+      key: 'description' + idCode
+    },
+    {
+      title: 'Example',
+      dataIndex: 'example',
+      key: 'example'  + idCode
+    },
+    {
+      title: 'Data type',
+      dataIndex: 'dataType',
+      key: 'dataType' + idCode
+    },
+    {
+      title: 'Required',
+      dataIndex: 'required',
+      key: 'required' + idCode
+    },
+  ];
+
+  const getTitle = () => {
+    switch (`${entity}-${dataField}`) {
+      case 'application-clientConfig':
+        return t('InformationModal.title.application.clientConfig');
+      case 'application-layerTree':
+        return t('InformationModal.title.application.layerTree');
+      case 'application-layerConfig':
+        return t('InformationModal.title.application.layerConfig');
+      case 'application-toolConfig':
+        return t('InformationModal.title.application.toolConfig');
+
+      case 'layer-clientConfig':
+        return t('InformationModal.title.layer.clientConfig');
+      case 'layer-sourceConfig':
+        return t('InformationModal.title.layer.sourceConfig');
+      case 'layer-features':
+        return t('InformationModal.title.layer.features');
+
+      case 'user-details':
+        return t('InformationModal.title.user.details');
+      case 'user-clientConfig':
+        return t('InformationModal.title.user.clientConfig');
+      case 'user-providerDetails':
+        return t('InformationModal.title.user.providerDetails');
+
+      default:
+        return [{
+          propertyName: 'TODO',
+          description: 'TODO',
+          example: 'TODO',
+          dataType: 'TODO',
+          required: 'TODO',
```

Comment:
There are some `TODO`s left.

Reply:
These are not TODOs, actually. They are just fallback values. But I'll return `undefined` instead in order to skip showing the table as a whole.